web-registry/src/pages/CreditClassDetails/CreditClassDetails.AdditionalInfo.tsx

```diff
@@ -38,39 +47,62 @@ const AdditionalInfo = <T extends CreditClassMetadataLD>({
   const unknownFields = getClassUnknownFields(metadata);
 
   return (
-    <Box sx={{ py: 8 }}>
-      <Grid container spacing={8}>
-        <MetaDetail
-          label="credit type"
-          value={capitalizeWord(creditTypeName)}
-        />
-        <MetaDetail label="registry" value={sourceRegistry} />
-        <MetaDetail
-          label="carbon offset standard"
-          value={carbonOffsetStandard}
-        />
-        <ApprovedMethodologiesList
-          methodologyList={metadata['regen:approvedMethodologies']}
-        />
-        <MetaDetail label="project activities" value={projectActivities} />
-        <MetaDetail label="sectoral scopes" value={sectoralScopes} />
-        <MetaDetail label="Tokenization Source" value={tokenizationSource} />
-        <MetaDetail label="ecosystem type" value={ecosystemTypes} />
-        <MetaDetail label="verification method" value={verificationMethod} />
-        <MetaDetail label="co-benefits" value={coBenefits} />
-        <MetaDetail label="measured GHGs" value={measuredGHGs} />
-        <BufferPoolAccounts
-          bufferPoolAccounts={bufferPoolAccounts?.['schema:itemListElement']}
-        />
-        {unknownFields.map(([fieldName, value]) => (
+    <Box
+      sx={{
+        p: 8,
+        backgroundColor: 'primary.main',
+        border: '1px solid',
+        borderColor: 'info.light',
+        borderRadius: '0 0 8px 8px',
+      }}
+    >
+      <Body size="lg">{CREDIT_CLASS_DETAILS_ADDITIONAL_INFO_HELPER_TEXT}</Body>
+      <Box sx={{ pt: 7 }}>
+        <Grid container spacing={8}>
+          <Grid item xs={12} sm={6} sx={{ flexDirection: 'column' }}>
+            <Box sx={{ display: 'flex', alignItems: 'center' }}>
+              <Label size="xs" sx={{ mr: 1 }}>
+                class id:
+              </Label>
+              <InfoTooltipWithIcon
+                title={CREDIT_CLASS_DETAILS_ADDITIONAL_INFO_CLASS_ID_TOOLTIP}
+                outlined
+              />
+            </Box>
+            <Body size="xl">{classId}</Body>
+          </Grid>
+          <MetaDetail
+            label="credit type"
+            value={capitalizeWord(creditTypeName)}
+          />
```

Comment:
It doesn't seem to be explicitly said in the issue description, but looking at the design, it seems like we could also remove this from the additional info section since we can see it at the top of the page. The only thing is that if there's no credit class image, then what do we do? I asked @erikalogie about that: https://www.figma.com/file/seUiQM2SPt78KJiDL0Ea8m?node-id=9031:253761&mode=design#509971709
so maybe let's keep this as is for now, and create a follow-up issue as soon as she's back and answers.

Reply:
so @erikalogie proposes to create a generic credit class img, meaning we can always show the credit type tag at the top left of it and remove it from the additional info. I think this is the best option, let's address this in a separate issue.

Reply:
Sounds good to me, @erikalogie or @blushi can one of you open an issue to track this?

Reply:
Sure, here it is: https://app.zenhub.com/workspaces/rnd-dev-team-5f8998bec8958d000f4609e2/issues/gh/regen-network/rnd-dev-team/1764